### PR TITLE
Create LIT6971YahegennaYaserat ...

### PR DESCRIPTION
**Pull request by CarstenHoffmannMarburg**
...  according to our discussion in https://github.com/BetaMasaheft/Documentation/issues/2520.

### CODE REVIEW

File: new/LIT6971YahegennaYaserat.xml [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+            </sourceDesc>
+        </fileDesc>
+        <encodingDesc>
+            <p>A digital born TEI file</p>
+        </encodingDesc>
+        <profileDesc>
+            <creation/>
+            <abstract>
+                <p>Amharic version of the <ref type="work" corresp="LIT1968Mashaf">Maṣḥafa śǝrʿat</ref> possibly also with comments
+                    similar to <ref type="work" corresp="LIT5933CommMuallaqa"/>.</p>
```
Comment (eu-genia) [inline new/LIT6971YahegennaYaserat.xml:36]: put the relation to `LIT1968Mashaf `also as a `relation`

File: new/LIT6971YahegennaYaserat.xml [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+                <language ident="am">Amharic</language>
+            </langUsage>
+        </profileDesc>
+        <revisionDesc>
+            <change who="CH" when="2024-04-26">Created entity</change>
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <div xml:lang="gez" type="edition">
```
Comment (eu-genia) [inline new/LIT6971YahegennaYaserat.xml:56]: am?

File: new/LIT6971YahegennaYaserat.xml [hunk 3]
```diff
@@ -0,0 +1,70 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT6971YahegennaYaserat" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="gez" xml:id="t1">Ya-ḥǝgǝnnā ya-śǝrʿat maṣḥaf</title>
```
Comment (thea-m) [inline new/LIT6971YahegennaYaserat.xml:8]: fidal to be added (and am)

**Review by thea-m — CHANGES_REQUESTED** [review]
Thank you! For future records, the file's IDs should be a bit shorter.

File: new/LIT6971YahegennaYaserat.xml (hunk 3 above)
Comment (eu-genia) [inline new/LIT6971YahegennaYaserat.xml:8]:
```
                <title xml:lang="am" xml:id="t1">......</title>
                <title xml:lang="gez" type="normalized" corresp="#t1">Ya-ḥǝgǝnnā ya-śǝrʿat maṣḥaf</title>
                <title xml:lang="en" xml:id="t2">........</title>
```

**Pull request merged**